src/redux/books/books.js

```diff
@@ -1,46 +1,43 @@
-import { addBookApi, getBookApi, removeBookApi } from '../../Api/booksApi';
-
-const ADD_BOOK = 'bookstore/books/ADD_BOOK';
-const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
-const GET_BOOK = 'bookstore/books/GET_BOOK';
-
-const initialState = [];
-
-export const addBook = (payload) => async (dispatch) => {
-  const book = { ...payload, item_id: payload.id };
-  await addBookApi(book);
-  dispatch({ type: ADD_BOOK, book });
-};
-
-export const removeBook = (id) => async (dispatch) => {
-  await removeBookApi(id);
-  dispatch({ type: REMOVE_BOOK, id });
-};
-
-export const getBook = () => async (dispatch) => {
-  const data = await getBookApi();
-  const books = Object.keys(data).map((id) => {
-    const book = data[id][0];
-    book.id = id;
-    return book;
-  });
-  dispatch({ type: GET_BOOK, books });
-};
-
-const reducer = (state = initialState, action) => {
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD = 'bookstore/Book/ADD';
+const REMOVE = 'bookstore/Book/REMOVE';
+
+const initialState = [
+  {
+    id: uuidv4(),
+    title: 'The Hunger Game',
+    author: 'Suzanne Collins',
+  },
+  {
+    id: uuidv4(),
+    title: 'Dune',
+    author: 'Frank Herbert',
+  },
+  {
+    id: uuidv4(),
+    title: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+  },
+];
+
+export function addBook(book) {
+  return { type: ADD, book };
+}
+
+export function removeBook(id) {
+  return { type: REMOVE, id };
```

Comment:
- As per the previous reviewer Kindly use  redux toolkit for this project as per the requirements, you can go [here](https://redux-toolkit.js.org/introduction/getting-started) to learn more about it